netci.groovy

```diff
@@ -276,6 +276,44 @@ def buildArchConfiguration = ['Debug': 'x86',
 }
 
 // **************************
+// Define outerloop testing for netfx that can build and run.  Run locally on each machine.
+// **************************
+[true, false].each { isPR ->
```

Comment:
I'll go over this tomorrow morning because I have some questions to do what you are suggesting